tck/src/main/java/org/eclipse/microprofile/openapi/tck/ModelConstructionTest.java

```diff
@@ -460,6 +460,23 @@ public void openAPITest() {
 
         Tag otherTag = createConstructibleInstance(Tag.class);
         checkListImmutable(o, OpenAPI::getTags, otherTag);
+
+        // reset the list
+        o.removeSecurityRequirement(sr);
+        o.removeSecurityRequirement(sr2);
+
+        // ensure that scope is compatible with a couple of schemes where compatibility was added in 3.1.0
+        final SecurityRequirement sr3 = createConstructibleInstance(SecurityRequirement.class);
+        sr3.addScheme("mutualTLS", "scope");
+        final SecurityRequirement sr4 = createConstructibleInstance(SecurityRequirement.class);
+        sr4.addScheme("HTTP", "scopeTwo");
+        checkSameObject(o, o.addSecurityRequirement(sr3));
+        checkSameObject(o, o.addSecurityRequirement(sr4));
+        assertEquals(o.getSecurity().size(), 2, "The list is expected to contain two entry.");
+        checkListEntry(o.getSecurity(), sr3);
+        checkListEntry(o.getSecurity(), sr4);
+        o.removeSecurityRequirement(sr3);
+        o.removeSecurityRequirement(sr4);
```

Comment:
Not sure this test is appropriate here, I think we should move it to the `securityRequirementTest` method in the same class (which currently only adds custom schemes with no scopes).

Reply:
I've moved this down to `securityRequirementTest`.